designs/panorama_plugin/panorama_plugin.md

```diff
@@ -0,0 +1,53 @@
+# Panorama Security Plugin
+This is a security appliance plugin for the Panorama Security Management Appliance, which orchestrates PaloAlto virtual firewalls deployed by OSC. The plugin shall conform to the [security manager api](https://github.com/opensecuritycontroller/security-mgr-api) specification.
+
+## Assignees
+Dmitry Gerenrot ([dmitryintel](https://github.com/dmitryintel)).
+
+## Constraints and Assumptions
+The Panorama version is 7.1. The firewalls are running PAN-VM-KVM-7.1.4.
+
+## Design Changes
+
+We propose the following workflow for creating a security group from OSC.
+
+- Begin by creating a distributed appliance on OSC with PAN-VM-KVM-7.1.4 as the service function. OSC then creates a Device Group under panorama via Panorama API. (For example, **DistributedAppGroup**).
+- Proceed by defining the security policies on panorama as follows:
+	- For each policy to be managed by the **Panorama Security Plugin**, there will be a dynamic address group (DAG) under the **DistributedAppGroup** and a _shared_ tag, e.g., **OSC_Policy_1**. Panorama administrators _create these manually_.
+	- Under the panorama **Policies** tab, create a security policy to target the DAG as usual.
+	
+![](./images/AddressGroups.PNG)
+
+- There should be no more manual steps on the panorama side.
+- In the OSC, create a security group. (E.g. security group id is **aaaaa01**). For each address in a security group, the OSC will create an address object under the **DistributedAppGroup**.
+- OSC also creates a tag of the form  **OSC_SecurityGroup_aaaaa01**.
+- There will be one such tag per security group. All the addresses within the security group will be labeled by it.
```

Comment:
clarify that the tags are on panorama.

All the addresses within the security group will be ~~labeled~~ tagged by the custom osc tag "OSC_SecurityGroup_aaaaa01" on panorama.